nodebb

```diff
@@ -458,8 +460,23 @@ var commands = {
 
 			async.series([
 				function (next) {
+					if (global.env === 'development') {
+						return next();
+					}
+
+					cproc.exec('git pull', {
+						cwd: __dirname,
+						stdio: [0, 1, 2],
+					}, next);
```

Comment:
I believe we'll want to keep `git pull` out of here, as different installs might not use git.